### DISCUSSION TITLE
Conditional content display

### DISCUSSION THREAD

**Discussion by amigthea** [elementor/elementor#21641] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I need to change my website logo on specific days and would like to automate that

### Describe the solution you'd like

I would like to schedule a different logo image based on date

**Comment by acornavi** [comment]
This is an essential functionality. Please check the e-addons display widget.

**Reply by amigthea** [reply]
so that's not an elementor addons but a third party plugin? I would rather have it integrated in elementor but, for the sake of testing, can you please link me that addon?

**Reply by acornavi** [reply]
I know. I shared it as an excellent example of how it can be done. https://e-addons.com
It is free
